src/tools/kotlin-entity-generator.ts

```diff
@@ -239,15 +131,43 @@ ${expression}
     const fieldCount = Object.keys(this.node.schema.fields).length;
     const blocks: string[] = [];
 
+    const fieldVals: string[] = [];
+    for (const {field, type, isCollection, escaped, defaultVal, nullableType} of this.schemaDescriptor.fields) {
+      if (this.opts.wasm) {
+        // TODO: Add support for collections in wasm.
```

Comment:
Can you link to a bug / issue?

Reply:
Well, I am skeptical of filing bugs for the functionality that we are not sure we want to implement in the next couple of quarters, I think it obfuscates things by drowning real problems around issues/bugs that are not relevant.